nf_core/modules/info.py

```diff
@@ -51,9 +56,9 @@ def init_mod_name(self, module):
             ).unsafe_ask()
             if local:
                 if self.repo_type == "modules":
-                    modules = self.module_names["modules"]
+                    modules = self.get_modules_clone_modules()
                 else:
-                    modules = self.module_names.get(self.modules_repo.fullname)
+                    modules = self.modules_json.get_all_modules().get(self.modules_repo.fullname)
```

Comment:
This line throws an error when running `modules info` because there is no `modules_json` attribute. I believe this is because `init_mod_name` is called in line 39 and `modules_json` is not initialised until line 41.

Reply:
Yes you are right. Nice catch!